policykit/policyengine/views.py

```diff
@@ -611,25 +612,96 @@ def clean_up_proposals(action, executed):
 
 @csrf_exempt
 def initialize_starterkit(request):
-    from policyengine.models import StarterKit, CommunityPlatform
-
-    starterkit_name = request.POST['starterkit']
-    community_name = request.POST['community_name']
-    creator_token = request.POST['creator_token']
-    platform = request.POST['platform']
-
-    starter_kit = StarterKit.objects.get(name=starterkit_name, platform=platform)
-
-    community = CommunityPlatform.objects.get(community_name=community_name)
-    starter_kit.init_kit(community, creator_token)
-
-    logger.info('starterkit initialized')
-
-    redirect_route = request.GET.get("redirect")
-    if redirect_route:
-        return redirect(f"{redirect_route}?success=true")
+    """
+    Takes a request object containing starter-kit information.
+    Initializes the community with the selected starter kit.
+    """
+    from policyengine.models import Proposal, CommunityPlatform, Policy, CommunityRole, CommunityUser
+
+    post_data = json.loads(request.body)
+
+    logger.debug(f'Initializing with starter kit: {post_data["starterkit"]}')
+    f = open(f'{os.getcwd()}/starterkits/{post_data["starterkit"]}.txt')
+
+    kit_data = json.loads(f.read())
+
+    community = CommunityPlatform.objects.get(community_name=post_data["community_name"])
```

Comment:
The `community_name` is not necessarily unique, we should probably use primary key instead. No need to change it in this PR, though, since that was already an issue.

Reply:
Ah, true! Left a comment there